README.md

```diff
@@ -1,32 +1,34 @@
 [![Build Status](http://jenkins.njoy21.io:8080/buildStatus/icon?job=constants)](http://jenkins.njoy21.io:8080/job/constants/)
 
 # constants
-This repository contains a collection of mathematical and physical constants. 
+This repository contains a collection of [mathematical](src/constants/math/README.md) and [physical](src/constants/physics/README.md) constants. Additional information about the constants provided&mdash;and their values&mdash; see the associated README files.
 
-**Note:** For the constants given here, their values are actually given in three places in this repository:
+**Note:** For the mathematical constants, their values are actually given in three places in this repository:
 
-1. Here in the `README.md` file,
-2. In the header file that defines the value for C++, and
-3. In the files used to test/verify the value.
+1. in the associated `README.md` files;
+2. in the header file that defines the value for C++; and
+3. in the files used to test/verify the values.
 While every effort is made to ensure that each of these values are the same, inconsistencies may appear. If discovered, please [file an issue](https://github.com/njoy/constants/issues).
 
-## Mathematical Constants
-The (short) list of mathematical constants was taken from [Wikipedia:Mathematical constant](https://en.wikipedia.org/wiki/Mathematical_constant). While there are many more constants given there, we have limited this initially to the [Basic mathematical constants](https://en.wikipedia.org/wiki/Mathematical_constant#Basic_mathematical_constants).
-
-  | Name                      | Variable name | Value               | Reference                                                 | Reference Date | 
-  | ----                      | ------------- | -----               | ---------                                                 | 
-  | Archimedes constant (π)   | `pi`          | `3.141592653589793` | [http://mathworld.wolfram.com/Pi.html]()                  | 2018-09-13     | 
-  | Euler's number (e)        | `e`           | `2.718281828459045` | [http://mathworld.wolfram.com/e.html]()                   | 2018-09-13     | 
-  | Pythagoras' constant (√2) | `root2`       | `1.414213562373095` | [http://mathworld.wolfram.com/PythagorassConstant.html]() | 2018-09-13     | 
-
-### Using mathematical constants
 
+### Using the constants
+Here is an example of how the mathematical constants can be used.
 ```cpp
 #include "constants/math.hpp"
 
 double r{1.0};
 
 auto area = njoy::constants::math::pi*r**2;
+
+#include "constants/physics/CODATA2014.hpp"
+
+auto CODATA = njoy::constants::CODATA2014;
```

Comment:
CODATA looks like a class

Reply:
They look like class names because they start with a capital letter. However, the Style Guide (Section 4) prefers the exception to "standard conventions". Since CODATA is a known acronym, this is the preferred way to call the variable.